modules/contact/src/actions/ContactAction.C

```diff
@@ -1028,6 +1071,129 @@ ContactAction::addNodeFaceContact()
   }
 }
 
+void
+ContactAction::createSidesetPairsFromGeometry()
+{
+  mooseInfo("The contact action is reading the list of sidesets and matching them assuming that: "
+            "1) Those sidesets wrap around the surfaces of the bodies, 2) Axisymmetry of the "
+            "geometry and the mesh exists. Deviations from axisymmetry may give rise to wrong "
+            "contact pairing. The user is encouraged to check the screen output to confirm pairs "
+            "were assigned correctly.");
+
+  _mesh = &_problem->mesh();
+
+  if (!_mesh)
+    mooseError("Internal error in contact action.");
```

Comment:
This is probably an error that will never be hit, but I'd still say what actually happened: "Failed to obtain mesh for automatically generating contact pairs"